### PR TITLE
Add recipe for Sylius Settings Plugin by @monsieurbiz

### PR DESCRIPTION
**Pull request by jacquesbh**
| Q             | A
| ------------- | ---
| License       | MIT

https://github.com/monsieurbiz/SyliusSettingsPlugin

### CODE REVIEW

File: monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+monsieurbiz_sylius_settings_admin:
+    resource: "@MonsieurBizSyliusSettingsPlugin/Resources/config/routes/admin.yaml"
+    prefix: /%sylius_admin.path_name%
```
Comment (Nyholm) [inline monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml:3]: Will this really work? 
Excuse my ignorance, but I was sure you could not add parameters like this in the routes.

**Review by Nyholm — COMMENTED** [review]
Just a small question to confirm

File: monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml (hunk 1 above)
Comment (jacquesbh) [reply monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml:3]: It works pretty well indeed. We have the same in production.

The parameter is defined by Sylius itself here:

https://github.com/Sylius/Sylius/blob/f27be445c11eccd0556d5edbafd60067960ca46c/src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml#L38

This is actually very convenient to secure the admin path with a random name!

Comment (jacquesbh) [reply monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml:3]: @Nyholm What do you think?

Comment (jacquesbh) [reply monsieurbiz/sylius-settings-plugin/1.0/config/routes/monsieurbiz_settings_plugin.yaml:3]: ping @Nyholm ? :p

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Could you tell me why Travis is red?

**Comment by jacquesbh** [comment]
> Could you tell me why Travis is red?

Hey @Nyholm!

I'm pretty sure it has to do with the installation process.
As a Sylius plugin you install it on top of a Sylius installation, which requires the  sylius/ui-bundle.
The ui bundle adds it's own configuration `sylius_ui`: https://github.com/Sylius/SyliusUiBundle/blob/b34241bb76415faff996b27dc123077b8389778d/DependencyInjection/Configuration.php#L24

We check our recipe on a Sylius standard edition here: https://github.com/monsieurbiz/SyliusSettingsPlugin/runs/2415461391?check_suite_focus=true
You can get the details in the step "Install plugin".

My understanding is that there is no auto configuration for the sylius/ui-bundle so it's not declared in the `bundles.php`, therefor there is no way a default symfony app would work with our declaration of some new blocks.

Maybe we could require the sylius/ui-bundle, that could solve the issue?
But it's inside the sylius/sylius… https://github.com/Sylius/Sylius/blob/c68ccd7fc7bda5f4dfac279f15bf669b23c37100/composer.json#L189

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**